workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -91,7 +93,10 @@ def run(self, mount_root, tmp_dir):
                 except Exception as limited_storages_ex:
                     Logger.warn('Unable to parse CP_CAP_LIMIT_MOUNTS value({}) with error: {}.'.format(limited_storages, str(limited_storages_ex.message)), task_name=self.task_name)
 
-            nfs_count = len(filter((lambda ds: ds.storage.storage_type == NFS_TYPE), available_storages_with_mounts))
+            cloud_region = self._get_cloud_region()
+            nfs_count = len(filter((lambda dsm: dsm.storage.storage_type == NFS_TYPE
```

Comment:
Maybe it is a better approach to filter NFS storages by region when we are loading them, to completely exclude them from mount process?